ARAnalytics.m

```diff
@@ -530,17 +530,16 @@ + (void)error:(NSError *)error withMessage:(NSString *)message
 
 + (void)pageView:(NSString *)pageTitle
 {
-    if (!pageTitle) return;
-    
-    [_sharedAnalytics iterateThroughProviders:^(ARAnalyticalProvider *provider) {
-        [provider didShowNewPageView:pageTitle];
-    }];
+    [self pageView:pageTitle withProperties:nil];
```

Comment:
My thinking was that the problem with using this is that it's an API breaking change as some providers might be overriding `-pageView:` which would not be called anymore when calling `-pageView:withProperties:`, but I did not actually check the provider implementations yet, so it might be moot.